MAGIC2.cfg

```diff
@@ -23,9 +23,10 @@ photon_delay = 13    % ns
 % ------------------------- Optical parameters --------------------------
 
 parabolic_dish      = 1        % Intermediate shape (mirrors on parabolic dish).
-focal_length        = 1697     % Effective focal length for camera.
+focal_length        = 1700     % Focal length for camera.
+effective_focal_length = 1821.2 % corrected for 1.0713 aberration 
 mirror_focal_length = 0        % Adapted automatically unless specified in file
-dish_shape_length   = 1697   % If using a single fixed focal length for a genuine parabolic dish
+dish_shape_length   = 1700   % If using a single fixed focal length for a genuine parabolic dish
```

Comment:
From the docs:

>  DISH_SHAPE_Length
> (type: Double, items: 1, default: "0", minimum: "0", maximum: "10000", units:
> centimeters)
> Dish curvature length, best equal to focal length. For a Davies-Cotton dish, this is
radius of the sphere on which the mirror tiles are positioned. For a parabolic dish,
this is the focal length of the paraboloid on which the mirrors are placed. This is not
normally needed but only when variations to the standard shapes are tried out, e.g.
intermediate shapes between parabolic and Davies-Cotton. For the solid secondary
mirror optics (mirror class 2), this parameter does not apply. Unit: cm. Normally 0
to adapt it to overall focal length.

So I think we can just remove this?

Reply:
Yes, it won't make a difference if you set it to zero, 1700 or remove it completely. We generally prefer to be explicit in the configuration files though.

Reply:
I would not remove it for MAGIC, unless it is also removed for LST. The reason is that the configs that are read earlier stay unless they are overwritten by new settings, so we would then have dish_shape_length for MAGIC that is inherited from LST, which obviously is very misleading.
Actually I realized that the same happened for the effective focal length, so thanks a lot @orelgueta and @maxnoe for pointing that this parameter is missing in the MAGIC part! 

If no comments come I will merge the PR in the afternoon

Reply:
makes, sense! +1 for leaving it at 1700 then